docs/querying/operators.md

```diff
@@ -192,7 +192,7 @@ vector of fewer elements with aggregated values:
 * `count_values` (count number of elements with the same value)
 * `bottomk` (smallest k elements by sample value)
 * `topk` (largest k elements by sample value)
-* `quantile` (calculate φ-quantile (0 ≤ φ ≤ 1) over dimensions)
+* `quantile` (calculate φ-quantile (0 ≤ φ ≤ 1) over dimensions; use 0.5 to get the median)
```

Comment:
This should be be no more than an aside, it shouldn't clutter up the main table.

Reply:
OK, but you're not telling me where to put it. Could you tell me where you want it?

As an alternative, considering it's 3-4 words, you can make the edit directly and close this.